AdyenCard/Components/Card/CardScannerController.swift

```diff
@@ -62,14 +66,36 @@ internal protocol CardScannerControlling {
                 .failure(CardScannerError.scanningError)
             }
         }
+
+        private func makeNavigationController() -> UINavigationController {
+            guard #available(iOS 13.0, *) else { return UINavigationController() }
+
+            let appearance = UINavigationBarAppearance()
+            appearance.configureWithDefaultBackground()
+
+            let navigationController = UINavigationController()
+            navigationController.navigationBar.standardAppearance = appearance
+            navigationController.navigationBar.compactAppearance = appearance
+            navigationController.navigationBar.scrollEdgeAppearance = appearance
+
+            return navigationController
+        }
+
+        private func makeCancelBarButton() -> UIBarButtonItem {
+            UIBarButtonItem(barButtonSystemItem: .cancel, target: self, action: #selector(handleCardScanningCancelation))
+        }
+
+        @objc private func handleCardScanningCancelation(sender: NSObject) {
+            presenter.navigationController?.topViewController?.dismiss(animated: true)
+        }
     }
 
 #else // canImport(AdyenCardScanner)
 
     internal final class CardScannerController: CardScannerControlling {
         internal var isScannerAvailable: Bool { false }
         internal var onScanComplete: ((Result<CardModel, any Error>) -> Void)?
-        internal func openCardScanner() {}
+        internal func openCardScanner(title: String?) {}
```

Comment:
I'd suggest `openCardScanner(titled: )` or `openCardScanner(with title)`

Reply:
I prefer any of these options.

Reply:
Other thought - this could just be a property on the `CardScannerController` 🤷

Reply:
- `titled` is not aligned with the common API design approach we use across the codebase
- `with title` converts into openCardScanner(with: "Scan Your Card") which has poor clarity
So I'd go with a simple variable. The sacrifice is that one can forget to set it, but at least it follows the `UIViewController` approach